test/ServerInvoiceTest.php

```diff
@@ -69,26 +74,26 @@ function testBasic()
 
         # Retrieve each invoice by id
         $invoice = new Invoice();
-        $invoice->invoiceId =  $invoices[0]->invoiceId;
+        $invoice->setInvoiceId($invoices[0]->getInvoiceId());
         $invoice = $invoice->get(false, $this->tenant->getTenantHeaders());
         $this->assertNotEmpty($invoice);
-        $this->assertNotEmpty($invoice->accountId);
-        $this->assertNotEmpty($invoice->invoiceId);
-        $this->assertNotEmpty($invoice->currency);
-        $this->assertEquals($invoice->amount, 0);
-        $this->assertEquals($invoice->balance, 0);
-        $this->assertEmpty($invoice->items);
+        $this->assertNotEmpty($invoice->getAccountId());
+        $this->assertNotEmpty($invoice->getInvoiceId());
+        $this->assertNotEmpty($invoice->getCurrency());
+        $this->assertEquals($invoice->getAmount(), 0);
+        $this->assertEquals($invoice->getBalance(), 0);
+        $this->assertEmpty($invoice->getItems());
 
         $invoice = new Invoice();
-        $invoice->invoiceId =  $invoices[1]->invoiceId;
+        $invoice->setInvoiceId($invoices[1]->getInvoiceId());
         $invoice = $invoice->get(true, $this->tenant->getTenantHeaders());
         $this->assertNotEmpty($invoice);
-        $this->assertNotEmpty($invoice->accountId);
-        $this->assertNotEmpty($invoice->invoiceId);
-        $this->assertNotEmpty($invoice->currency);
-        $this->assertEquals($invoice->amount, 500);
-        $this->assertEquals($invoice->balance, 0);
-        $this->assertNotEmpty($invoice->items);
-        $this->assertEquals(1, count($invoice->items));
+        $this->assertNotEmpty($invoice->getAccountId());
+        $this->assertNotEmpty($invoice->getInvoiceId());
+        $this->assertNotEmpty($invoice->getCurrency());
+        $this->assertEquals($invoice->getAmount(), 500);
```

Comment:
phpunit throws an error here: 

```
Failed asserting that 500 matches expected 150.0.
```

But this might actually be correct, maybe the test case is just wrong here?